### PR TITLE
Merge Team Roles & Structures docs into a single unified source of structure :)

### PR DESCRIPTION
**Pull request by daviddias**
- [x] merge Roles & Structures doc into one
- [x] augment definition of roles
- [x] communicate who is playing the BDFL, LEAD and COORDINATOR roles for the project

### CODE REVIEW

File: TEAMS_ROLES_STRUCTURES.md [hunk 1]
```diff
@@ -1,38 +1,48 @@
...
 
 ## Description
 
 The IPFS Project is developed by multiple teams from multiple organizations and both intermittent and continuous invidividual contributors. A structure was developed (and continues to evolve) around Working & Research Groups, OKRs, and PM best practices that were designed to support the already large and growing group of people to coordinate among each other at a global scale.
 
-This document captures the Team Structures that emerged and that led to the many of the sucesses within the IPFS project, namely: Working Groups & Research Groups. At the end of this document, you will find pointers to Teams and Projects that have grown to become their own entities, namely: IPLD, libp2p and Multiformats.
+This document presents the the existing teams that emerged and that led to the many of the sucesses within the IPFS project, their structure(namely: Working Groups & Research Groups) and a definition of roles existing in the IPFS org. At the end of this document, you will find pointers to Teams and Projects that have grown to become their own entities, namely: IPLD, libp2p and Multiformats.
 
-#### One line definition
+## Team Structures
```
Comment (daviddias) [inline TEAMS_ROLES_STRUCTURES.md]: @momack2 probably it is time to describe also what a Tiger Team is given that those have become more and more used.

Comment (momack2) [reply TEAMS_ROLES_STRUCTURES.md]: Added!

File: TEAMS_ROLES_STRUCTURES.md [hunk 2]
```diff
@@ -50,25 +61,154 @@ Some great examples from this structure are: research-pubsub which led to the im
...
+- Guide and Gatekeeper of the Working Group Roadmap
+- Decision disambiguator for the technical matters of the Working Group
+
+### Working Group Technical Project Manager (TPM)
+
+The Technical Project Manager (sometimes referenced as Project Manager, Program Manager, and cat herder) is a team enabler. They own the Quarterly Planning process including OKRs and Retrospectives. They ensure that the coordination strategy the WG selected is well executed (weekly syncs, taking notes, communicating needs to other WGs).
+
+The Technical Project Manager should also have a role on contributing to the projects the Working Group is tackling and have an understanding of the technical challenges and needs from the team.
+
+It happens that some working groups are ok with having the same person playing the Captain + Technical Project Manager role together, it works for when the Working Group is nascent or the team is small.
```
Comment (olizilla) [inline TEAMS_ROLES_STRUCTURES.md]:
```suggestion
It happens that some working groups have the same person playing both the Captain and Technical Project Manager role. This should be limited to situations where the Working Group is nascent or the team is small.
```

**Comment by daviddias** [comment]
Thank you all for reviews and improvements! Meeerging 🚢

**Pull request merged**